### DISCUSSION TITLE
Contributor Call: 10 December 2022

### DISCUSSION THREAD

**Discussion by l3xander** [freesewing/freesewing#3108] [Support] [answered]
You are all invited to the FreeSewing Contributor Call.
Please leave a comment here or [on Discord](https://discord.freesewing.org/) to add things to the agenda.
This issue will be updated after the call with notes and to-dos.

### About
FreeSewing's contributor call is an informal hangout/chat/meeting that is open to all FreeSewing contributors and users alike.
The call is organized/hosted by @tangerineshark and @kaermorhan.
The call does not have a fixed structure or agenda items. Everyone is free to suggest agenda items, ask questions, or bring ideas.

**Details**
Location: https://discord.freesewing.org (meeting room: 🔊`voice chat`)
Date: 10 December 2022 **5pm** UTC, which can be found in your local time zone [here](https://savvytime.com/converter/utc-to-belgium-antwerpen-ca-los-angeles-il-chicago-ny-new-york-city-brazil-sao-paulo-united-kingdom-london-france-paris-south-africa-johannesburg-msd-india-mumbai-hkt-japan-tokyo-australia-sydney/dec-10-2022/5pm).
Or check the bot in Discord! `#meetings`
The meetings will always be at 6pm, Antwerp time.
⚠There have been timezone switches lately, careful!

### Agenda
- How to document freesewing.org-specific things in the core reference documentation
- Managing site dependencies in the monorepo (possible auto-generating of `package.json` files for the sites)
- RFC: Using [AWS Fargate](https://aws.amazon.com/fargate/) for hosting our backend
- Wanted: Drop-shippers for patron gifts (locations: US, Canada, UK, Australia) 
- Holiday break - no call on 24/12 - next call on 07/01
- ...

> Notes will be added after the call.

**Comment by eriese** [comment]
Can we add pull request checks to the agenda? Which ones do we like, which ones can we maybe ditch?

**Comment by KaerMorhan** [comment, marked as answer]
**Notes**

Participants:

- @tangerineshark
- @lexander2002
- @KaerMorhan
- @joostdecock
- @bobgeorgethe3rd
- @BenJamesBen
- @eriese
- @starfetch 
- @SeaZeeZee

✨ **Joost**
Documenting FS-specific things in the core reference documentation.
There are plans to add a lot of configuration that will be visible or not visible depending on the wish of the user. It is technically not core reference documentation but it needs to be documented. Where do we put it?
_Enoch: I say we put it in the core library and put a note near it stating that it needs to be implemented._
_Joost: I was thinking of the similar thing, creating a page explaining how to use it. So it will be put into the documentation but it will be stated that it’s extra._

✨ **Joost & Enoch**
Managing site dependencies in the monorepo.
There are problems of dependencies between different versions in the development. Some things will run smoothly but others will require to run “the whole monorepo” (a lot of stuff).
We could do a better job at documenting all of the dependencies so new developers will have a resource to turn to.
We will add it to the same file we use to avoid everything else in the monorepo.

✨ **Enoch**
Pull-request checks.
Some pull requests don’t run because we don’t want them to run and then we get a red x near it and then nobody looks at what actually went wrong and then it just gets to be postponed or pushed further into development. The list is growing and it is not controlled.
_Joost: I have recently tried to run what needs to be run and not run what doesn’t need to be run. I disabled dependobot from being run: there is a specific code that allows to ignore curtain builds. Now we need to find a way to ignore a certain build without counting it as a failure._

✨ **Joost**
RFC: Using AWS Fargate for hosting our backend.
So we will not have a server, Amazon will just run a container for us. For users it will make no difference, it’s on the agenda to have more people involved if they want to.

✨ **Zwaluw**
I was trying to help a friend to make a Hi shark and I was struggling with the documentation. Could we make something to make them more accessible? Maybe making levels for the documentation? Not for a complete beginner but also not for people who already know a lot about sewing.
_Joost: Documentation differs wildly. Sometimes there is no documentation, sometimes there is no documentation and people come in to make quick bullet points or write documentation with illustrations, and sometimes there is documentation from the designer._
_Zwaluw: How are we managing handling of documentation/sewing instructions. We could ask for feedback from users and decide more specific things after having results._
_Joost: We also had an idea of having people who would take a pattern under their wing (pattern shepherds) so they can make sure it works on different versions, write documentation etc. We also could have a person who will make sure every pattern has a shepherd and pop up with “do you want to adopt this pattern” messages._

✨ **Karen**
Wanted: Drop-shippers for patron gifts.
We have gifts for our patrons and it’s hard to ship them directly from Joost’s house so we need shipping buddies.

Announcement: there will be no CC on 24/12, the next one will be on 07/01.

✨ **Benjamin**
Are we agreeing that shoulder point, shoulder tip and shoulder edge are the same?
_Zwaluw: Yes, they are._
_Joost: We need to make it consistent so it’s easier to follow. (after brief voting) Tip it is._
_Benjamin: I will check the documentation again._

**Comment by joostdecock** [comment]
Closing to give room to active discussions
